assets/js/modules/analytics/components/common/PropertySelect.js

```diff
@@ -93,7 +93,12 @@ export default function PropertySelect() {
 						value={ id }
 						data-internal-id={ internalWebPropertyId } // eslint-disable-line sitekit/acronym-case
 					>
-						{ name }
+						{ sprintf(
+							/* translators: %1$s: property name, %2$s: property ID */
+							__( '%1$s (%2$s)', 'google-site-kit' ),
+							name,
+							id
+						) }
```

Comment:
This looks great for the most part, except we should only do this for actual properties. We don't want to format the value in this way for `property_create` since this is an internal detail.

![image](https://user-images.githubusercontent.com/1621608/115865206-b109be00-a440-11eb-8dfa-e65dd9496c15.png)

The easiest solution here is to use `internalWebPropertyId` to conditionally render with:
* If truthy, format the value with the name and ID, 
* if there is no internal web property ID, only output the name